src/transaction/log_replication_atomic.hpp

```diff
@@ -107,6 +110,7 @@ namespace cublog
        * Since multiple DDL operations can be executed within single transaction,
        * more than one classes can be mapped to one transaction */
       std::multimap <TRANID, OID> m_locked_classes;
+      std::unordered_map <OID, std::string> m_classname_map;
```

Comment:
It looks like problems may arise if multiple ddl statements are performed on the same table in one transaction. Do you also have a verification scenario for this case?

Reply:
There is no problem I found. 
Only the names of the class 
1) at the time of lock acquisition (m_classname_map) and 
2) at the time of commit are needed. 

Even if multiple `LOG_SCHEMA_MODIFICATION_LOCK`s are replicated, the name stored in m_classname_map is from the moment of the first lock acquisition, so it doesn't matter.

I have tested various DDL operations within a single transaction and confirmed that there are no problems.
- CREATE - RENAME - DROP
- RENAME - RENAME - RENAME
- RENAME - RENAME - DROP
- CREATE - RENAME - RENAME